server/src/main/resources/db/changelog/20191009160315-remove-duplicate-indexes.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~  Copyright (c) 2009 - ${YEAR} Red Hat, Inc.
```

Comment:
Copyright statement should have the ${year} filled in.

Reply:
I have updated the statement.

Reply:
What Barnaby meant is that this should literally be the actual year that this file was created, as most source code files have in Candlepin, instead of a variable: ```Copyright (c) 2009 - 2019 Red Hat, Inc.```